### DISCUSSION TITLE
Apply for FLOSS Fund

### DISCUSSION THREAD

**Discussion by kskarthik** [globaleaks/globaleaks-whistleblowing-software#4317] [General]
Hello!

Globaleaks could apply to https://floss.fund/ which grants $1 million / year to FOSS projects globally

**Comment by evilaliv3** [comment]
Thank you @kskarthik 

We noticed this and we are evaluating.

At the moment considering the capacity of the fund for this year (1 million) we did not consider applying to enable projects at prototype stage to have their chance.
